README.md

```diff
@@ -1 +1,26 @@
-# volume-microservice
\ No newline at end of file
+# volume-microservice
+
+[![Python 3.9.2](https://img.shields.io/badge/python-3.9.2-blue.svg)](https://www.python.org/downloads/release/python-392/)
+
+A project by [Cornell AppDev](http://cornellappdev.com), a project team at Cornell University.
```

Comment:
Can we add a description of its relation to volume-backend?

Reply:
You got it